File: src/codes/Users_Manual/gain/focus_1.cs
~using AUTD3Sharp;
~using AUTD3Sharp.Utils;
~using AUTD3Sharp.Gain;
~var x = 0.0f;
~var y = 0.0f;
~var z = 0.0f;
var g = new Focus(new Point3(x, y, z))
            .WithIntensity(0xFF)
            .WithPhaseOffset(0x00);

File: src/codes/Users_Manual/gain/focus_1.rs
# use autd3::prelude::*;
# #[allow(unused_variables)]
# fn main() {
# let x = 0.;
# let y = 0.;
# let z = 0.;
let g = Focus::new(Point3::new(x, y, z))
            .with_intensity(0xFF)
            .with_phase_offset(0x00);
# }